### DISCUSSION TITLE
sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.

### DISCUSSION THREAD

**Discussion by LuanShiZhiJian** [apache/skywalking#11470] [Q&A]
JAVA VERSION:11
SKYWALKING:9.6.0

![image](https://github.com/apache/skywalking/assets/41558924/e53bea9c-b314-45c1-bbf5-aee03bf17315)

HOW TO SOLVE IT?THANKS!

**Comment by wu-sheng** [comment]
This is IP binding failing. It means another service has used this IP + port.

**Reply by wu-sheng** [reply]
I can't see what your title means. It doesn't make sense at all.

**Reply by LuanShiZhiJian** [reply]
It is error from starting the Skywalking service.

**Reply by LuanShiZhiJian** [reply]
What is the default port for this service in Windows?
